### PR TITLE
Release article for image templates page

### PR DESCRIPTION
**Pull request by bgeuken**
https://build.opensuse.org/image_templates

### CODE REVIEW

**Comment by bgeuken** [comment]
Should be merged together with https://github.com/openSUSE/open-build-service/pull/3090

**Comment by bgeuken** [comment]
@openSUSE/open-build-service Please anyone review and then merge. Thanks:-)

File: _posts/releases/2017-05-09-new-image-templates-page.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+And there it is!
+
+<img src="/images/posts/image_templates_template_page.png" style="margin:20px;" title="OBS Image Templates Page">
+
+The image template page is a place where you can find all kinds of pre-configured KIWI
+image configurations provided by other users and projects.
+
+If you are just about to create a new image for your server to host your new
+shiny app. Or you want to create a pre-configured openSUSE Leap 42.2 for your
+laptop. You might want to have a look at this page.
```
Comment (mlschroe) [inline _posts/releases/2017-05-09-new-image-templates-page.md]: There's something wrong with the '.' in the above paragraph. Do you mean something like "Say you are just about... Then you might want to..."?

File: _posts/releases/2017-05-09-new-image-templates-page.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Maybe someone has already done the job!
+
+Once you have found what you were looking for, select the template and click on the
+'Create appliance' button.
+This will branch the image configuration into your personal OBS project. Where you
+can customize it.
```
Comment (mlschroe) [inline _posts/releases/2017-05-09-new-image-templates-page.md]: Same as above, I guess. "Where you can customize it" is a bit short. Maybe just leave out the '.'?

**Comment by mlschroe** [comment]
See in-commit comments. Other that that it's fine with me. Thanks!

**Comment by bgeuken** [comment]
@mlschroe Thanks much! I've updated it.

File: _posts/releases/2017-05-11-new-image-templates-page.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+And there it is!
+
+<img src="/images/posts/image_templates_template_page.png" style="margin:20px;" title="OBS Image Templates Page">
+
+The image template page is a place where you can find all kinds of pre-configured KIWI
+image configurations provided by other users and projects.
+
+Are you just about to create a new image for your server to host your new
+shiny app? Or you want to create a pre-configured openSUSE Leap 42.2 for your
```
Comment (Ana06) [inline _posts/releases/2017-05-11-new-image-templates-page.md]: shouldn't this be:

Or **do** you want to create a pre-configured openSUSE Leap 42.2 for your laptop?

File: _posts/releases/2017-05-11-new-image-templates-page.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+laptop? Then you might want to have a look at the new image template page.
```
Comment (Ana06) [inline _posts/releases/2017-05-11-new-image-templates-page.md]: I think *may* would be better that *might* here: http://writingexplained.org/may-vs-might-difference

Then you *may* want to have a look at the new image template page

File: _posts/releases/2017-05-11-new-image-templates-page.md (hunk 3 above)
Comment (bgeuken) [inline _posts/releases/2017-05-11-new-image-templates-page.md]: I've changed it

File: _posts/releases/2017-05-11-new-image-templates-page.md (hunk 4 above)
Comment (bgeuken) [inline _posts/releases/2017-05-11-new-image-templates-page.md]: I prefer the old version

**Comment by bgeuken** [comment]
Michael approved this PR via IRC this morning and the OBS PR got merged now.

We are ready to go :fireworks:

**Pull request merged**